wallet-new/integration/Functions.hs

```diff
@@ -436,27 +437,33 @@ runAction wc action = do
 
             -- Check the transaction.
             log $ "postTransaction: " <> show newPayment
-            result  <-  respToRes $ postTransaction wc newPayment
+            newTx  <-  respToRes $ postTransaction wc newPayment
+
+            let sumCoins f =
+                    sum . map (getCoin . unV1 . pdAmount) . toList $ f newTx
```

Comment:
You can add `V1 . mkCoin .` here if possible since we are not interested in the old datatypes (you convert them later).

Reply:
it is added later https://github.com/input-output-hk/cardano-sl/pull/2766/files#diff-4ea23860944836eb6f9e56fa8200d0b6R452 - after some arithmetic operations becouse `V1` is missing `Num` instance